cloverage/test/cloverage/coverage_test.clj

```diff
@@ -449,3 +450,26 @@
                       require (constantly nil)]
           (t/is (= {:errors 3}
                    ((cov/runner-fn (merge opts runner-opts)) []))))))))
+
+(t/deftest test-coverage-under?
```

Comment:
Always glad to meet fellow `are`rs 😄

While, merely aesthetical, you might enjoy this version as well:
```clj
(t/deftest test-coverage-under?
  (t/testing "check that the different thresholds work properly"
    (let [coverage-under? #'cov/coverage-under?
          forms           {}]
      (with-redefs [rep/total-stats (constantly {:percent-lines-covered 97
                                                 :percent-forms-covered 75})]
        (t/are [result fail-threshold line-fail-threshold form-fail-threshold]
            (= result (coverage-under? forms fail-threshold line-fail-threshold form-fail-threshold))
          #_result #_fail-threshold #_line-fail-threshold #_form-fail-threshold ;; Non-zero fail-threshold
          true     100              0                     0                     ; line and form coverage both under fail-threshold
          true     90               0                     0                     ; line coverage is under fail-threshold, form coverage is above fail-threshold
          false    70               0                     0                     ; line and form coverage both above fail-threshold
          false    70               100                   0                     ; line-fail-threshold ignored because fail-threshold is non-zero
          false    70               0                     100                   ; form-fail-threshold ignored because fail-threshold is non-zero
          false    70               100                   100                   ; line- and form-fail-threshold ignored because fail-threshold is non-zero
          ; fail-threshold is 0
          nil      0                0                     0
          false    0                90                    70                    ; line coverage is above line-fail-threshold and form coverage is above form-fail-threshold
          true     0                100                   70                    ; line coverage is under line-fail-threshold, form coverage is above form-fail-threshold
          true     0                90                    90                    ; line coverage is above line-fail-threshold, form coverage is under form-fail-threshold
          true     0                100                   100)))))              ; line coverage is under line-fail-threshold and form coverage is under form-fail-threshold
```

IME it makes it friendly to a greater variety of people. The alignment is achieved with clojure-mode's clojure-align (and I add the `#_` column headers by hand)

Reply:
Cool, I like the column header and alignment, it's much more readable, I applied the change. Thanks for all the reviews!